webapp/components/survey/SurveyForm/nodeDefs/nodeDefUIProps.js

```diff
@@ -132,7 +132,20 @@ const getProp = (prop, defaultValue) => R.pipe(NodeDef.getType, getPropByType(pr
 
 export const getIconByType = getPropByType('icon')
 
-export const getInputTextProps = getProp('inputText', { mask: false })
+export const getDecimalProps = (inputTextProps) => (nodeDef) => ({
+  ...inputTextProps,
+  mask: InputMasks.decimalLimited(16, Number(NodeDef.getProp(NodeDef.propKeys.maxNumberDecimalDigits)(nodeDef) || 6)),
```

Comment:
maybe you can introduce a NodeDef.getmaxNumberDecimalDigits function that defaults to 6

Reply:
not sure about this, because not all the nodeDefs have this, what do you think=?

Reply:
I think it's fine here

Reply:
I'm not sure if it's a good idea to use the propKeys outside of the NodeDef itself, I would prefer to have a specific function to access that prop.